conda/recipes/cucim/build.sh

```diff
@@ -25,7 +25,7 @@ cp -P python/install/lib/* python/cucim/src/cucim/clara/
 
 pushd python/cucim
 
-echo "PYTHON: ${PYTHON}"
-$PYTHON setup.py install
+echo "PYTHON: ${PYTHON_VER}"
+$PYTHON_VER setup.py install
```

Comment:
Should this still be `$PYTHON` as it is referring to the executable and not the version?